cores/arduino/stm32/usb/cdc/usbd_cdc_if.c

```diff
@@ -245,6 +247,11 @@ void CDC_deInit(void)
   }
 }
 
+uint8_t CDC_connected()
+{
+  return hUSBD_Device_CDC.dev_state == USBD_STATE_CONFIGURED && hostPortOpen;

```

Comment:
I will use directly `lineState`here:
```suggestion
  return hUSBD_Device_CDC.dev_state == USBD_STATE_CONFIGURED && lineState;
```